custom-images/generate_custom_image.py

```diff
@@ -46,7 +46,7 @@
 
 # Old style images: 1.2.3
 # New style images: 1.2.3-deb8
-_VERSION_REGEX = re.compile(r"^\d+\.\d+\.\d+(-.{4})?$")
+_VERSION_REGEX = re.compile(r"^\d+\.\d+\.\d+(-.{3,4})?(-.{4})?$")
```

Comment:
replace `(-.{3,4})?` with `(-RC\d+)?` to be more explicit?

Reply:
ok~XD